modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -336,7 +336,7 @@ interface ObjectValidation {
 	lineCount?: number;
 	name: LocalizedValue<string>;
 	objectValidationRuleSettings?: {
-		name: 'objectFieldExternalReferenceCode';
+		name: 'outputObjectFieldExternalReferenceCode';
```

Comment:
@naraandrade Can we make it a constant?
It's used in three differrent places.

Reply:
@mateussgn Unfortunately, you won't be able to call the constant here because the TypeScript will interpret it as a type and not a value. So please, keep this as it is.